tests/sklearn/test_sklearn_autolog.py

```diff
@@ -1462,3 +1462,35 @@ def test_metric_computation_handles_absent_labels():
     assert not metrics
     assert tags == get_expected_class_tags(model)
     assert MODEL_DIR in artifacts
+
+
+@pytest.mark.parametrize(
+    "cross_val_func_name", ["cross_validate", "cross_val_score", "cross_val_predict"]
+)
+def test_autolog_disabled_on_sklearn_cross_val_api(cross_val_func_name):
+    mlflow.sklearn.autolog()
+    from sklearn import linear_model
+
+    def assert_autolog_disabled_during_exec_cross_val_fun(run_):
+        params, metrics, tags, artifacts = get_run_data(run_.info.run_id)
+        assert params == {} and metrics == {} and tags == {} and artifacts == []
+
+    diabetes = sklearn.datasets.load_diabetes()
+    X = diabetes.data[:150]
+    y = diabetes.target[:150]
+    lasso = linear_model.Lasso()
+
+    cross_val_func = getattr(sklearn.model_selection, cross_val_func_name)
+    with mlflow.start_run() as run:
+        cross_val_func(lasso, X, y, cv=3)
+        assert_autolog_disabled_during_exec_cross_val_fun(run)
+
+    exp_id = mlflow.tracking.fluent._get_experiment_id()
+    runs_info = mlflow.list_run_infos(exp_id)
+    cross_val_func(lasso, X, y, cv=3)
+    runs_info2 = mlflow.list_run_infos(exp_id)
+
+    # assert before cross_val_func executing and after cross_val_func executed,
+    # mlflow run info counts are the same, which proves that cross_val_func executing
+    # does not generate new mlflow runs.
+    assert len(runs_info) == len(runs_info2)
```

Comment:
Can we simply say `Ensure cross_val_func doesn't create a new run`?

```suggestion
    # Ensure `cross_val_func` doesn't create a new run
    exp_id = mlflow.tracking.fluent._get_experiment_id()
    runs_info = mlflow.list_run_infos(exp_id)
    cross_val_func(lasso, X, y, cv=3)
    runs_info2 = mlflow.list_run_infos(exp_id)
    assert len(runs_info) == len(runs_info2)
```